x-pack/plugins/security_solution/public/detections/components/alerts_table/index.tsx

```diff
@@ -360,7 +359,7 @@ export const AlertsTableComponent: React.FC<AlertsTableComponentProps> = ({
   const casesPermissions = useGetUserCasesPermissions();
   const CasesContext = kibana.services.cases.getCasesContext();
 
-  if (loading || indexPatternsLoading || isEmpty(selectedPatterns)) {
+  if (loading || isEmpty(selectedPatterns)) {
```

Comment:
This condition was making the tGrid unmount while loading dataView patterns after creating/editing/removing a runtime field, causing the request to be incorreclty aborted. Removing this condition here has no implication since the `indexPatternsLoading` value is also taken into account inside the tGrid.